CHANGELOG.md

```diff
@@ -19,10 +19,13 @@
      scroll the actual error message off the screen.
     * Exceptions occurring in tab completion functions are now printed to stderr before returning control back to
     readline. This makes debugging a lot easier since readline suppresses these exceptions.
+    * Added support for custom Namespaces in the argparse decorators. See description of `ns_provider` argument
```

Comment:
This new feature would benefit from a mention in the Sphinx docs section about the argparse decorators and an example.

Reply:
Done